scripts/helper.py

```diff
@@ -110,10 +110,6 @@ def get_markdown_row(field, link, multi_field):
         # Add ticks around examples to not break table
         example = "`{}`".format(field["example"])
 
-    # If link is true, it link to the anchor is provided. This is used for the use-cases
```

Comment:
I don't think `helper.py` is referenced anywhere in the repo other than `use-cases.py`.

We may be able to remove the file entirely?

Reply:
Good catch! Removed